app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -213,17 +278,33 @@ private fun RecentlyVisitedCaption(count: Int) {
     )
 }
 
+/**
+ * Recent items divider.
+ *
+ * @param modifier [Modifier] allowing to perfectly place this.
```

Comment:
```suggestion
 * @param modifier Optional [Modifier] to be applied to the layout.
```
Using one of your previous kDoc description so we hopefully don't keep rewording this differently everywhere.

Reply:
I would't say this is optional.
Though about even not providing a default here to force users pass a Modifier.
Saw there are differences in how this composable should be placed depending on which type of history is shown so while a Modifier could control a lot of different options that comment should hint to the users that it is intended to help place this appropriately.